.github/workflows/ci.yml

```diff
@@ -0,0 +1,324 @@
+name: Tests master
+
+on:
+  # This avoids having duplicate builds for a pull request
+  push:
+    branches:
+    - master
+  pull_request:
+    branches:
+    - master
+
+jobs:
+  ############################################################################
+  # Lint jobs
+  ############################################################################
+  lint-py:
+    name: Lint Python
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false 
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Cache pip
+        uses: actions/cache@v1
+        with:
+          path: ~/.cache/pip
+          key: linux-py38-lint-pip-${{ hashFiles('**/setup.py') }}
+          restore-keys: |
+            linux-py38-lint-pip-
+      - name: Setup Python
+        uses: actions/setup-python@v1
+        with:
+          python-version: '3.8'
+          architecture: 'x64'
+      - name: Install python dependencies
+        run: python -m pip install --editable .[test]
+      - name: Show python environment
+        run: |
+          python --version
+          python -m pip list
+      - name: Run python lint
+        run: pylint lektor
+
+  lint-js:
+    name: Lint JS
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch
+        uses: actions/checkout@v2
+      - name: Cache node
+        uses: actions/cache@v1
+        with:
+          path: ~/.npm
+          key: linux-node13-lint-${{ hashFiles('**/lektor/admin/package-lock.json') }}
```

Comment:
I think it's fine to drop the `-lint` part. That way the linux-node tests can re-use the same cache.

Also `node13` -> `node-13` to ensure other jobs can re-use it.

Reply:
Fixed!